src/Generating/FinishGen.cpp

```diff
@@ -15,6 +15,7 @@
 #include "../Simulator/FireSimulator.h"
 #include "../World.h"
 #include "../IniFile.h"
+#include "../BiomeDef.h"
```

Comment:
It *could* be possible that this is also included in the globals. Did you try to compile it without this include?